modules/markup/html.go

```diff
@@ -317,48 +317,52 @@ func RenderEmoji(
 	return ctx.postProcess(rawHTML)
 }
 
-var byteBodyTag = []byte("<body>")
-var byteBodyTagClosing = []byte("</body>")
-
 func (ctx *postProcessCtx) postProcess(rawHTML []byte) ([]byte, error) {
 	if ctx.procs == nil {
 		ctx.procs = defaultProcessors
 	}
 
-	// give a generous extra 50 bytes
-	res := make([]byte, 0, len(rawHTML)+50)
-	res = append(res, byteBodyTag...)
-	res = append(res, rawHTML...)
-	res = append(res, byteBodyTagClosing...)
-
 	// parse the HTML
-	nodes, err := html.ParseFragment(bytes.NewReader(res), nil)
+	nodes, err := html.ParseFragment(bytes.NewReader(rawHTML), nil)
```

Comment:
Need to force the html fragment to be part of a body:

```suggestion
	nodes, err := html.ParseFragment(bytes.NewReader(rawHTML), &html.Node{
		Type: html.ElementNode,
		DataAtom: atom.Body,
		Data: "body",
	})
```